libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1231,14 +1302,17 @@ AP_InertialSensor::_init_gyro()
     Vector3f new_gyro_offset[INS_MAX_INSTANCES];
     float best_diff[INS_MAX_INSTANCES];
     bool converged[INS_MAX_INSTANCES];
+#if HAL_INS_TEMPERATURE_CAL_ENABLE
+    float start_temperature[INS_MAX_INSTANCES];
```

Comment:
Needs to be 
```
float start_temperature[INS_MAX_INSTANCES] {};
```
to appease the compiler gods

Reply:
its dynamically allocated, so guaranteed zero, the init just wastes flash

Reply:
ahh, you're right, it's stack allocated, fixed